nccs/pipeline/direct/agriculture.py

```diff
@@ -69,6 +71,109 @@ def get_impf_set_tc():
+
+    Based on the JRC publication: https://publications.jrc.ec.europa.eu/repository/handle/JRC105688
+
+    Regional functions available for Africa, Europe, North America, and Asia
+
+    Other regions (Oceania and South America will get the Global function assigned)
+
+    In this study the damage fractions in the damage curves are intended to span from zero
+    (no damage) to one (maximum damage).
+
+    For agriculture the damage is related to a loss in output when the yield is destroyed by floods.
+
+
+
+    """
+
+    # Use the flood module's lookup to get the regional impact function for the country
+    country_info = pd.read_csv(RIVER_FLOOD_REGIONS_CSV)
+    impf_id = country_info.loc[country_info['ISO'] == country_iso3alpha, 'impf_RF'].values[0]
+
+    if impf_id == 1:
+        impf_jrc_africa = ImpactFunc(
+            id = 1,
+            name = "Flood Africa JRC Agriculture",
+            intensity_unit="m",
+            haz_type="RF",
+            intensity = np.array([0, 0.5, 1, 1.5, 2, 3, 4, 5, 6]),
+            mdd = np.array([0.00, 0.24, 0.47, 0.74, 0.92, 1.00, 1.00,
+                            1.00, 1.00]),
+            paa = np.array([1, 1, 1, 1, 1, 1, 1,
+                            1, 1])
+        )
+        impf_jrc_africa.check()
+        imp_fun_set = ImpactFuncSet([impf_jrc_africa])
+
+    elif impf_id == 2:
+        impf_jrc_asia = ImpactFunc(
+            id = 1,
+            name = "Flood Asia JRC Agriculture",
+            intensity_unit="m",
+            haz_type="RF",
+            intensity = np.array([0, 0.5, 1, 1.5, 2, 3, 4, 5, 6]),
+            mdd = np.array([0.00, 0.14, 0.37, 0.52, 0.56, 0.66, 0.83,
+                            0.99, 1.00]),
+            paa = np.array([1, 1, 1, 1, 1, 1, 1,
+                            1, 1])
+        )
+        impf_jrc_asia.check()
+        imp_fun_set = ImpactFuncSet([impf_jrc_asia])
+
+    elif impf_id == 3:
+        impf_jrc_europe = ImpactFunc(
+            id = 1,
+            name = "Flood Europe JRC Agriculture",
+            intensity_unit="m",
+            haz_type="RF",
+            intensity = np.array([0, 0.5, 1, 1.5, 2, 3, 4, 5, 6]),
+            mdd = np.array([0.00, 0.30, 0.55, 0.65, 0.75, 0.85, 0.95,
+                            1.00, 1.00]),
+            paa = np.array([1, 1, 1, 1, 1, 1, 1,
+                            1, 1])
+        )
+        impf_jrc_europe.check()
+        imp_fun_set = ImpactFuncSet([impf_jrc_europe])
+
+    elif impf_id == 4:
+        impf_jrc_northamerica = ImpactFunc(
+            id = 1,
+            name = "Flood North America JRC Agriculture",
+            intensity_unit="m",
+            haz_type="RF",
+            intensity = np.array([0, 0.5, 1, 1.5, 2, 3, 4, 5, 6]),
+            mdd = np.array([0.02, 0.27, 0.47, 0.55, 0.60, 0.76, 0.87,
+                            0.95, 1.00]),
+            paa = np.array([1, 1, 1, 1, 1, 1, 1,
+                            1, 1])
```

Comment:
Our analyses are going to look wrong with an impact function that has impacts when the hazard is zero. Let's add an extra point to the function here so that the curve goes through (0, 0).
```suggestion
            intensity = np.array([0, 0.01, 0.5, 1, 1.5, 2, 3, 4, 5, 6]),
            mdd = np.array([0, 0.02, 0.27, 0.47, 0.55, 0.60, 0.76, 0.87,
                            0.95, 1.00]),
            paa = np.array([1, 1, 1, 1, 1, 1, 1, 1,
                            1, 1])
```